### DISCUSSION TITLE
Navigation auto hide panels

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#47] [Q&A] [answered]
Do we need a function to auto hide or hide / show on touch the **navigation panels**?
(all other screen elements / buttons already have auto hide settings)

Could be useful in OLED displays to prevent burn-in or when want to see more map.
(and not use the screen saver)

- Navigation panels can auto hide with a timer (like everything else)
- They can appear before turns and in voice announcements
(like when the screen saver activates the screen)
- Or manually: touch screen > panels hide > touch screen > panels appear

**Comment by DolomitesTom** [comment]
You mean with an own timer for auto hide or the same as for the buttons? When I think about, yes this could be very useful.  But they should appear again when tapping, so a combination between appear on announcements and tapping. The more I think about, the more I am excited. You would then have the full screen for the map. Fantastic.

**Comment by WolfgangZ42** [comment, marked as answer]
May be interesting for someone - not for me. I would not use it, because I'll not wait (or tap) for the display of essential information and with transparency I have already a fullscreen map:
<img src="https://user-images.githubusercontent.com/121806238/220377032-98620ca4-690d-4d85-b7fc-6b37664bfae5.png" width="400"/>
